src/modules/acbs/acbs.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { Controller, Get } from '@nestjs/common';
+
+import { AcbsAuthenticationService } from './acbs-authentication.service';
+
+// TODO API-70: Delete this controller
```

Comment:
I've only added this controller temporarily to allow us to manually test the token. When we have added an endpoint that uses the token then we should delete this controller

Reply:
it feels like this should be a helper function rather than endpoint? this if we need to execute the functionality, but not expose it to users

Reply:
The endpoint is only temporary and is to allow for manual testing of the functionality until a real endpoint (i.e. API-70) is merged.

If we don't think we need/want that then I can revert the commit that adds the controller and exposes the data as an endpoint

Reply:
To explain a bit more, I'd expect API-70 to use the `AcbsAuthenticationService` in some form, and not use this `TestController` at all

Reply:
ok cool. happy with that